matlab_simulation/02-examples_practice/gyroCovEstimation/gyroCovEstimation.m

```diff
@@ -0,0 +1,110 @@
+%% A Matlab program to calculate the convariances from a Gyroscope Data.
```

Comment:
This is a lecture example, not a practice problem example, move to the other examples folder.